### PR TITLE
Quiz draft

### PR DESCRIPTION
**Pull request by jamierashley**
### What is a quick description of the change?

First draft of quiz with pagination and minor css

### Screenshot/Screengrab after this change?

Screenshot 2020-07-14 at 11 54 32 AM

### Is this fixing an issue?

none

### Are there more details that are relevant?

Questions can be changed just basic for now

### CODE REVIEW

**Comment by aliciasimon** [comment]
LGTM

**Review by aliciasimon — APPROVED** [review]

**Comment by LilCSharp** [comment]
I agree with the front-end part of this, but the objective of the quiz is not to do filtering. It is supposed to bias qualities in the database. We will have to talk more about this in the stand-up

**Comment by jamierashley** [comment]
> I agree with the front-end part of this, but the objective of the quiz is not to do filtering. It is supposed to bias qualities in the database. We will have to talk more about this in the stand-up

What part of this do you think is relevant to filtering? These questions aren't set they can be changed. And almost no backend for the quiz has been written because I'm waiting on your database as these qualities correspond with a user email.

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -18,21 +18,196 @@ var arrayOfProducts = Array();
...
+//filtering by tags
+function gluten(){
+    let str = " ";
+    var checkBox = document.getElementById("gluten free");
+    if(checkBox.checked == true){
+        for(i = 0; i <  arrayOfProducts.length; i++){
+           let prodTags = arrayOfProducts[i].tag_list;
+           let prodBrand = arrayOfProducts[i].brand;
+           let valName = document.getElementById("gluten free").value;
+           let prodName = arrayOfProducts[i].name;
```
Comment (LilCSharp) [inline src/main/webapp/script.js:185]: There is a way to do the filtering without all of these separate tags in one function. I would not suggest doing it in this way.

**Review by LilCSharp — COMMENTED** [review]
I think that there is a better way to deal with the filtration. I have a method that can integrate well with this system, but we should talk about this later

**Review by LilCSharp — APPROVED** [review]

**Pull request closed without merging**